themes/liquidprompt/liquidprompt.theme.bash

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# Wrapper to use liquidprompt with bashit
+
+targetdir="$BASH_IT/themes/liquidprompt/liquidprompt"
+gray="\[\e[1;90m\]"
+
+cwd="$PWD"
+if cd "$targetdir" &>/dev/null && git rev-parse --is-inside-work-tree &>/dev/null; then
+    true
+else
+    git clone https://github.com/nojhan/liquidprompt.git "$targetdir" && \
```

Comment:
This creates a directory `$BASH_IT/themes/liquidprompt/liquidprompt`, which is then shown as untracked to the Bash-it Git repo.

Can you please add a `.gitignore` file in `themes/liquidprompt`, which ignores the nested `liquidprompt` folder?

```bash
┌─16-17:33[foo@foo-macbook:~/.bash_it] 1b9e2:con-f-use-master* ± 
└▪ git status
On branch con-f-use-master
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	themes/liquidprompt/liquidprompt/
```